libs/coin-modules/coin-polkadot/src/network/sidecar.ts

```diff
@@ -597,7 +634,8 @@ const getConstants = makeLRUCache(
  * @returns {Promise<Object>} consts
  */
 export const getTransactionMaterialWithMetadata = makeLRUCache(
-  async (): Promise<SidecarTransactionMaterial> => fetchTransactionMaterial(true),
+  async (currency: CryptoCurrency): Promise<SidecarTransactionMaterial> =>
+    fetchTransactionMaterial(currency, true),
   () => "polkadot",
```

Comment:
UP line 625`Store only one constants object since we only have polkadot.` Now we do have `westend` so we need to store it as well. Perhaps we can have 
`const getConstants = makeLRUCache(
  async (currency: CryptoCurrency): Promise<Record<string, any>> => fetchConstants(currency),
  () => currency.id ,
  hours(1, 1),
);`

Or something similar!

You will have to also modifie the `fetchConstants` function  in `./node/constants`